applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1112,6 +1091,48 @@ void UPwSmallStrainElement<TDim, TNumNodes>::InitializeBiotCoefficients(ElementV
     KRATOS_CATCH("")
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+std::vector<array_1d<double, TDim>> UPwSmallStrainElement<TDim, TNumNodes>::CalculateFluidFluxes(
+    const std::vector<double>& permeability_update_factors, const ProcessInfo& rCurrentProcessInfo)
+{
+    const GeometryType& rGeom      = this->GetGeometry();
+    const IndexType     NumGPoints = rGeom.IntegrationPointsNumber(this->GetIntegrationMethod());
+
+    std::vector<array_1d<double, TDim>> FluidFluxes;
+    ElementVariables                    Variables;
+    this->InitializeElementVariables(Variables, rCurrentProcessInfo);
+
+    const PropertiesType& rProp = this->GetProperties();
+
+    array_1d<double, TDim> GradPressureTerm;
+
+    // Create general parameters of retention law
+    RetentionLaw::Parameters RetentionParameters(this->GetProperties(), rCurrentProcessInfo);
+
+    for (unsigned int GPoint = 0; GPoint < NumGPoints; ++GPoint) {
+        this->CalculateKinematics(Variables, GPoint);
+        Variables.PermeabilityUpdateFactor = permeability_update_factors[GPoint];
+
+        GeoElementUtilities::InterpolateVariableWithComponents<TDim, TNumNodes>(
+            Variables.BodyAcceleration, Variables.NContainer, Variables.VolumeAcceleration, GPoint);
+        Variables.FluidPressure = CalculateFluidPressure(Variables);
+        RetentionParameters.SetFluidPressure(Variables.FluidPressure);
+
+        Variables.RelativePermeability =
+            mRetentionLawVector[GPoint]->CalculateRelativePermeability(RetentionParameters);
+
+        noalias(GradPressureTerm) = prod(trans(Variables.GradNpT), Variables.PressureVector);
+
```

Comment:
lines 1124 and 1126 are constructing the contents of the same variable. This interspacing contradicts that.

Reply:
Removed blank line